### PR TITLE
Fixed error for single channel input to the model

### PR DESCRIPTION
**Pull request by Medha-B**

### CODE REVIEW

File: src/main/java/org/sbml/spatial/segmentation/TrainUnetModel.java [hunk 1]
```diff
@@ -117,14 +116,15 @@ public static void main(String[] args) {
...
+			// ComputationGraph model = UNet.builder().updater(new
+			// Adam(1e-4)).build().init();
 			// ComputationGraph model = UNet.builder().updater(new Adam(new
 			// MapSchedule(ScheduleType.ITERATION, learningScheduleMap))).build().init();
 
 			// To change the number of input channels from 3 to 1
-			// ZooModel unet = UNet.builder().build();
-			// unet.setInputShape(new int[][]{{1, 128, 128}});
-			// ComputationGraph model = (ComputationGraph) unet.init();
+			ZooModel unet = UNet.builder().build();
```
Comment (tokkuman) [inline src/main/java/org/sbml/spatial/segmentation/TrainUnetModel.java]: I think it's best to set the optimizer to Adam, just as you did with 3 channels.
```
ZooModel unet = UNet.builder().updater(new Adam(1e-4)).build();
```

File: src/main/java/org/sbml/spatial/segmentation/TrainUnetModel.java [hunk 2]
```diff
@@ -21,15 +21,14 @@
 import org.deeplearning4j.ui.api.UIServer;
 import org.deeplearning4j.ui.model.stats.StatsListener;
 import org.deeplearning4j.ui.model.storage.InMemoryStatsStorage;
+import org.deeplearning4j.zoo.ZooModel;
 import org.deeplearning4j.zoo.model.UNet;
 import org.nd4j.linalg.api.ndarray.INDArray;
 import org.nd4j.linalg.dataset.api.iterator.DataSetIterator;
 import org.nd4j.linalg.dataset.api.preprocessor.DataNormalization;
 import org.nd4j.linalg.dataset.api.preprocessor.ImagePreProcessingScaler;
-import org.nd4j.linalg.learning.config.Adam;
```
Comment (tokkuman) [inline src/main/java/org/sbml/spatial/segmentation/TrainUnetModel.java]: You don't need to remove this line.

**Review by tokkuman — COMMENTED** [review]
It's better to use the same training conditions as the 3 channel model.

File: src/main/java/org/sbml/spatial/segmentation/TrainUnetModel.java (hunk 1 above)
Comment (Medha-B) [reply src/main/java/org/sbml/spatial/segmentation/TrainUnetModel.java]: Will make the necessary changes. Thanks.

**Comment by Medha-B** [comment]
Please check and let me know if this pull request is ready to be merged.

**Comment by tokkuman** [comment]
I'm sorry for the delay in checking. I think it's ok to merge!

**Comment by Medha-B** [comment]
Thanks for replying!

**Pull request merged**